setup.sh

```diff
@@ -448,6 +449,26 @@ function install_home()
   update_version
 }
 
+function show_current_env()
+{
+  local config_files_path="$etcdir"
+  local config_file_template="$config_files_path/strings/current_env_content.txt"
+  local shellrc="${HOME}/.bashrc"
+  local content_current_env_file
+
+  if [[ -f "$config_file_template" ]]; then
+    content_current_env_file="$(cat "${config_file_template}")"
+    if ! grep -Fxq "$content_current_env_file" "$shellrc"; then
```

Comment:
Run this command outside this `if` and use the full parameter name, e.g., use --fixed-strings instead of -F.